source/StigData/Processed/RHEL-7-3.8.xml

```diff
@@ -1954,2800 +1954,2641 @@ All=p+i+n+u+g+s+m+S+sha512+acl+xattrs+selinux
+      <DuplicateOf />
+      <IsNullOrEmpty>False</IsNullOrEmpty>
+      <LegacyId>V-72173</LegacyId>
+      <OrganizationValueRequired>False</OrganizationValueRequired>
+      <OrganizationValueTestString />
+      <RawString>Verify the operating system generates audit records when successful/unsuccessful attempts to use the "umount" command occur.
 
-/org/gnome/desktop/media-handling/automount-open
+Check that the following system call is being audited by performing the following series of commands to check the file system rules in "/etc/audit/audit.rules": 
 
-/org/gnome/desktop/media-handling/autorun-never
+$ sudo grep -w "/usr/bin/umount" /etc/audit/audit.rules
 
-If the output does not match the example, this is a finding.</RawString>
-    </Rule>
-    <Rule id="V-228563" severity="medium" conversionstatus="pass" title="SRG-OS-000480-GPOS-00227" dscresource="None">
-      <Description>&lt;VulnDiscussion&gt;If a world-writable directory is not owned by root, sys, bin, or an application User Identifier (UID), unauthorized users may be able to modify files created by others.
+-a always,exit -F path=/usr/bin/umount -F perm=x -F auid&gt;=1000 -F auid!=unset -k privileged-mount 
 
-The only authorized public directories are those temporary directories supplied with the system or those designed to be temporary file repositories. The setting is normally reserved for directories used by the system and by users for temporary file storage, (e.g., /tmp), and for directories requiring global read/write access.&lt;/VulnDiscussion&gt;&lt;FalsePositives&gt;&lt;/FalsePositives&gt;&lt;FalseNegatives&gt;&lt;/FalseNegatives&gt;&lt;Documentable&gt;false&lt;/Documentable&gt;&lt;Mitigations&gt;&lt;/Mitigations&gt;&lt;SeverityOverrideGuidance&gt;&lt;/SeverityOverrideGuidance&gt;&lt;PotentialImpacts&gt;&lt;/PotentialImpacts&gt;&lt;ThirdPartyTools&gt;&lt;/ThirdPartyTools&gt;&lt;MitigationControl&gt;&lt;/MitigationControl&gt;&lt;Responsibility&gt;&lt;/Responsibility&gt;&lt;IAControls&gt;&lt;/IAControls&gt;</Description>
-      <DuplicateOf />
-      <IsNullOrEmpty>False</IsNullOrEmpty>
-      <LegacyId>
-      </LegacyId>
-      <OrganizationValueRequired>False</OrganizationValueRequired>
-      <OrganizationValueTestString />
-      <RawString>The following command will discover and print world-writable directories that are not owned by a system account, assuming only system accounts have a UID lower than 1000. Run it once for each local partition [PART]:
+If the command does not return any output, this is a finding.</RawString>
+    </Rule>
+    <Rule id="V-204554" severity="medium" conversionstatus="pass" title="SRG-OS-000042-GPOS-00020" dscresource="None">
```

Comment:
This rule along with a handful of other ones was automated in the previous version of the STIG, any ideas what happened?

Reply:
V-204536 is another rule that was automated in pervious

Reply:
V-204538, V-204539, V-204542, V-204543, V-204544